components/suggest/src/schema.rs

```diff
@@ -51,11 +52,6 @@ pub const SQL: &str = "
         FOREIGN KEY(suggestion_id) REFERENCES suggestions(id) ON DELETE CASCADE
     );
 
-    CREATE TABLE pocket_custom_details(
```

Comment:
how does one do that 😅

Reply:
I believe `DROP TABLE IF EXISTS pocket_custom_details` should do.

After a second thought, since we're bumping the schema version as well, we will rebuild the entire db anyway. Maybe we don't need to drop anything. @linabutler Could you please hold me true? :)

Reply:
Nope, no need for a `DROP TABLE`. If we were doing a durable migration, where we wanted to preserve the database, we would—but the `1..10` branch in `upgrade_from` is replacing the entire database, so we can just delete this `CREATE TABLE` statement.